GliaWidgets/Sources/CallVisualizer/VideoCall/ViewController/VideoCallViewController.swift

```diff
@@ -32,6 +35,25 @@ public extension CallVisualizer {
         public override func loadView() {
             view = videoCallView
         }
+
+        public override func viewDidLoad() {
+            super.viewDidLoad()
+            proximityManager = .init(
+                view: self.view,
+                environment: .init(
+                    uiApplication: environment.uiApplication,
+                    uiDevice: environment.uiDevice,
+                    uiScreen: environment.uiScreen,
+                    notificationCenter: environment.notificationCenter
+                )
+            )
+            proximityManager?.start()
+        }
+
+        public override func viewWillDisappear(_ animated: Bool) {
+            super.viewWillDisappear(animated)
+            proximityManager?.stop()
```

Comment:
Do you think that there could be a situation where the view disappears and it appears again? Because `viewWillLoad` and `viewWillDisappear` are not equivalent. I don't know if it could go on `deinit` instead, or where could be a good place to put it. Someone help me :)

Reply:
Yes good point in the context of bubble appearing. The call is still on when bubble is present so it makes sense to move it to deinit

Reply:
fixed [648be6f](https://github.com/salemove/ios-sdk-widgets/pull/550/commits/648be6f90a46a7c5ccadf00f0514b099a48f6c73)